docs/en/stack/ml/dataframes.asciidoc

```diff
@@ -0,0 +1,57 @@
+[[ml-dataframes]]
+=== {dataframes-cap}
+
+{dataframes-cap} feature is available in 7.2 and later.
+
+A _{dataframe}_ is a transformation of a dataset by certain rules defined during
```

Comment:
I think you could consider dropping everything after "rules". i.e. In this high-level overview, I don't think it's necessary to specify *when* you define the rules.